docs/design/design-reviews/recipes/engineering-feasibility-spikes.md

```diff
@@ -0,0 +1,92 @@
+# Engineering Feasibility Spikes: identifying and mitigating risk
```

Comment:
I would recommend reworking this document so that it's as clear and concise as possible, integrating bullets and making it very action oriented. This would help make this more integrated and in line with the style of the rest of the content vs using more of a descriptive paragraph style.

Reply:
Thanks, @fnocera. I have updated content/document accordingly.